src/Interpreters/ErrorLog.cpp

```diff
@@ -74,8 +111,33 @@ void ErrorLogElement::appendToBlock(MutableColumns & columns) const
     columns[column_idx++]->insert(event_time);
     columns[column_idx++]->insert(code);
     columns[column_idx++]->insert(ErrorCodes::getName(code));
+    columns[column_idx++]->insert(error_time_ms / 1000);
+    columns[column_idx++]->insert(error_message);
     columns[column_idx++]->insert(value);
     columns[column_idx++]->insert(remote);
+    columns[column_idx++]->insert(query_id);
+
+    std::vector<UInt64> error_trace_array;
+    error_trace_array.reserve(error_trace.size());
+    for (auto * ptr : error_trace)
+        error_trace_array.emplace_back(reinterpret_cast<UInt64>(ptr));
+
+    columns[column_idx++]->insert(Array(error_trace_array.begin(), error_trace_array.end()));
+
+    #if defined(__ELF__) && !defined(OS_FREEBSD)
+    if (symbolize)
+    {
+        auto [symbols, lines] = generateArraysSymbolsLines(error_trace_array);
+
+        columns[column_idx++]->insert(symbols);
+        columns[column_idx++]->insert(lines);
+    }
+    else
+    #endif
```

Comment:
Just some nit picking. I think we usually don't indent preprocessor directives like this.
```suggestion
#if defined(__ELF__) && !defined(OS_FREEBSD)
    if (symbolize)
    {
        auto [symbols, lines] = generateArraysSymbolsLines(error_trace_array);

        columns[column_idx++]->insert(symbols);
        columns[column_idx++]->insert(lines);
    }
    else
#endif
```